operator/keys/rsa_linux.go

```diff
@@ -0,0 +1,103 @@
+//go:build linux
+
+package keys
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"math/big"
+
+	"github.com/microsoft/go-crypto-openssl/openssl"
+	"github.com/microsoft/go-crypto-openssl/openssl/bbig/bridge"
+)
+
+type privateKey struct {
+	privKey       *rsa.PrivateKey
```

Comment:
Do we need `privKey` in the linux implementation? For linux, we could create a new `*openssl.PrivateKeyRSA` when we create a new `privateKey`